tutorials/stackdriver-monitor-slow-query-mysql.md

```diff
@@ -0,0 +1,468 @@
+
```

Comment:
This file should be name `index.md`.
Take a look at a recent tutorial to see the expect structure: https://github.com/GoogleCloudPlatform/community/tree/master/tutorials/understanding-oauth2-and-deploy-a-basic-auth-srv-to-cloud-functions

Also, this file should have a specific header: see an example here: https://raw.githubusercontent.com/GoogleCloudPlatform/community/master/tutorials/understanding-oauth2-and-deploy-a-basic-auth-srv-to-cloud-functions/index.md

Reply:
I've change it